### ISSUE TITLE
Token issues

### ISSUE THREAD

**Issue by Lucas559-noob** [FoxsCode/DiscordPlus#1]
For a time using it, I got disconnected when I reloaded my tab, and after some minutes later, when It worked again, in servers, I was in a new server, and there some people asked "how I got here", "how I joined if there is nothing mentioning me that I joined", "since when I'm here", etc.
admins said that is strange, how people join without being pinged, then they thought was a raid, then someone said that this happened after installing this script, and later they started banning people who talked about those things.
My suspicious is:

- this get users token(searching for token, is possible to get a lot of results)
- and this make a user that got the token, join servers(how, idk) and try to spend their nitro(if have) to boost a server

And I have installed it a time ago, and the same thing happened, the only diference is that nothing worked on that

Also, first

**Comment by FoxsCode** [comment]
Hi Lucas, we were contacted about this on discord a while ago already, its been proven and confirmed that whoever did it wasnt for boosts, since 99% of the people that were botted to join seem to not have Nitro, so that wouldnt make any sense.
We can confirm that one of the scripts we used from other repository did seem to be infected but we are working on cleaning it already, thats why we took down the script from the Greasyfork site while we are at this, we dont want more users to be infected or damaged.

Also, you commented that you installed the script already some time ago, but that Discord++ you are talking about wasnt us but an impersonator trying to infect people with out idea. Back then we were working on this project and apparently one of the people who knew about it created that to infect people way before we started publishing the real Discord++, we even had to report it on two websites for it to be taken down (Yeah, thats why their's isnt up anymore, we had to report it)
For now we recommend to all of you to change your passwords and enable 2FA as a safety measure.
